universal-login-relayer/lib/config/config.prod.ts

```diff
@@ -6,6 +6,7 @@ export const config: Config =  Object.freeze({
   jsonRpcUrl: getEnv('JSON_RPC_URL', ''),
   port: getEnv('PORT', ''),
   privateKey: getEnv('PRIVATE_KEY', ''),
+  network: getEnv('NETWORK', ''),
   chainSpec: Object.freeze({
```

Comment:
idea: shell we add `name` to `chainspec` instead?

Reply:
👍  Already exists, so we should just set it from env in prod mode

Reply:
Cool, but we should add information in all the different files (test, dev).